src/main/java/pl/javastart/task/contracts/Contract.java

```diff
@@ -0,0 +1,44 @@
+package pl.javastart.task.contracts;
+
+public abstract class Contract implements Operations {
+
+    private int textsSent;
+    private int mmsSent;
+    private int callDurationInSeconds;
+
+    protected int getTextsSent() {
+        return textsSent;
+    }
+
+    protected void setTextsSent(int textsSent) {
+        this.textsSent = textsSent;
+    }
```

Comment:
To nie jest błąd i nie musisz tego zmieniać, ale gdybyś użył po prostu specyfikatora domyślnego, czy protected przy polach, to w klasach pochodnych kod byłby prostszy